Source/CombatExtended/CombatExtended/Loadouts/Dialog_ManageLoadouts.cs

```diff
@@ -73,7 +73,9 @@ public Dialog_ManageLoadouts(Loadout loadout)
             SetSource(SourceSelection.Ranged);
             doCloseX = true;
             closeOnClickedOutside = true;
-            closeOnEscapeKey = true;
+            closeOnCancel = true;
```

Comment:
Does the vanilla outfit manager have corresponding flags set?

Reply:
No Idea I just made an educated guess in most cases. In which class would that be?

Reply:
```csharp
// RimWorld.Dialog_ManageOutfits
public Dialog_ManageOutfits(Outfit selectedOutfit)
{
	this.forcePause = true;
	this.doCloseX = true;
	this.doCloseButton = true;
	this.closeOnClickedOutside = true;
	this.absorbInputAroundWindow = true;
	if (Dialog_ManageOutfits.apparelGlobalFilter == null)
	{
		Dialog_ManageOutfits.apparelGlobalFilter = new ThingFilter();
		Dialog_ManageOutfits.apparelGlobalFilter.SetAllow(ThingCategoryDefOf.Apparel, true, null, null);
	}
	this.SelectedOutfit = selectedOutfit;
}
```
For consistency, should keep the relevant options the same, so no close on cancel/accept.

Reply:
Changed.